index.html

```diff
@@ -224,5 +224,19 @@ <h3>Skills</h3>
             </div>
           </section>
             </main>
-</body>
-</html>
\ No newline at end of file
+            <footer>
+              <h2 class="contact-me">Contact me</h2>
+              <p class="footer-p">
+                If you have an application you are interested in developing,
+                 a feature that you need built or a project that needs coding.
+                  I’d love to help with it 
+              </p>
+              <form  class="feedback" action="https://formspree.io/html/" method="post">
+               <input class="name" type="text" placeholder="Enter your name" name="name" maxlength="30" required>
+               <input class ="email" type="email" placeholder="Enter your email" name="email" required>
+               <textarea rows="10" maxlength="500" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] As stated by the previous reviewer kindly add the required attribute for the Textarea field.